ocaml/idl/datamodel_errors.ml

```diff
@@ -706,6 +706,8 @@ let _ =
     ~doc:"The operation cannot be performed because this VDI has rrd stats" ();
   error Api_errors.vdi_too_small [ "vdi"; "minimum size" ]
     ~doc:"The VDI is too small. Please resize it to at least the minimum size." ();
+  error Api_errors.vdi_too_large [ "vdi"; "maximum size" ]
+    ~doc:"The VDI is too large. Cannot export to VHD >2TB" ();
```

Comment:
This is confusing. II talks about the VDI being too large and refers to "VHD > 2TB". It's not obvious what is too large. How about "Cannot export VDI larger than 2TB as a VHD" or similar.

Reply:
True. The last sentence should be deleted, this is a more generic error message.